controllers/mirroring/mirroring_controller.go

```diff
@@ -394,7 +392,7 @@ func (r *MirroringReconciler) reconcileBlockPoolMirroring(
 					)
 
 					cephBlockPool.Spec.Mirroring.Enabled = true
-					cephBlockPool.Spec.Mirroring.Mode = "image"
+					cephBlockPool.Spec.Mirroring.Mode = "init-only"
```

Comment:
Can we do this change as a post-upgrade operation? Do we know how would it effect existing RDR deployments?

Reply:
I am not sure how this change is going to affect upgrades. We would need to check that. 
@subhamkrai, do you know what will happen if we have blockpool mirroring enabled with mode as `image` and  then we change the mode to `init-only` and create the rns cr for implicit namespace to enable mirroring?

Reply:
I think changing mirroring mode is supported so we'll good